plugins/steve/resource-class.js

```diff
@@ -1337,7 +1337,7 @@ export default class Resource {
         const allOfResourceType = this.$rootGetters['cluster/all']( type );
 
         this.ownersByType[kind].forEach((resource, idx) => {
-          const resourceInstance = allOfResourceType.find(resource => resource?.metdata?.uid === resource.uid);
+          const resourceInstance = allOfResourceType.find(resource => resource?.metadata?.uid === resource.uid);
```

Comment:
This one will require closer review, as we're modifying an object property and need to name sure that the corresponding change `metdata` => `metadata` is in place.

Reply:
I've tried searching a couple of ways, and I can't find this _anywhere_ ...

There are 3 copies of this file in org:rancher
https://github.com/search?q=org%3Arancher+metdata&type=code
https://cs.github.com/?scopeName=All+repos&scope=&q=org%3Arancher+metdata

metadata-store is also occasionally misspelled:
https://cs.github.com/?scopeName=All+repos&scope=&q=rancher+metdata

But... not really sure how else to search...